hermetic_build/DEVELOPMENT.md

```diff
@@ -130,6 +130,9 @@ owl-bot copy-code --version
 The key step is `npm link`, which will make the command available in you current
 shell session.
 
+If you get a permission denied error when running the command `owl-bot`, try
+relinking owl-bot by running `npm unlink -g` and re-running the steps above.
```

Comment:
Why would rerun the same command can solve the permission issue?

Reply:
`npm link` has effect in the current shell session. My guess is that what's in an old session may get its permissions overriden.